osu.Game/Overlays/OptionsOverlay.cs

```diff
@@ -33,10 +33,13 @@ public class OptionsOverlay : OverlayContainer
 
         private ScrollContainer scrollContainer;
         private OptionsSidebar sidebar;
+        private OptionsSidebar.SidebarButton[] sidebarButtons;

```

Comment:
maybe find a way to move this array to be inside OptionsSidebar so we don't need this reference?